src/schema/artist/__tests__/index.test.js

```diff
@@ -737,4 +737,43 @@ describe("Artist type", () => {
       })
     })
   })
+
+  describe("filtered artworks", () => {
+    it("returns filtered artworks", () => {
+      const filterArtworksLoader = jest.fn().mockReturnValueOnce(
+        Promise.resolve({
+          hits: [
+            {
+              id: "im-a-cat",
+              title: "I'm a cat",
+              artists: ["percy"],
+            },
+          ],
+          aggregations: [],
+        })
+      )
+      rootValue.filterArtworksLoader = filterArtworksLoader
+
+      const query = `
+        {
+          artist(id: "percy") {
+            filtered_artworks(aggregations:[TOTAL], partner_id: null){
+              hits {
+                id
+              }
+            }
+          }
+        }
+      `
+
+      return runQuery(query, rootValue).then(
+        ({ artist: { filtered_artworks: { hits } } }) => {
+          expect(filterArtworksLoader.mock.calls[0][0]).not.toHaveProperty(
```

Comment:
Updated to use `jest.fn()`, and conveniently added this assertion as well (that `null` args made to the query were stripped out before being passed to the loader).